### DISCUSSION TITLE
Feature Request: support react-flow

### DISCUSSION THREAD

**Discussion by Kenzo-Wada** [mantinedev/mantine#6737] [Feature requests]
Hi, @rtivital !

Thank you for creating this awesome library!

I know you just got chart support in v7, but I personally use reactflow a lot, and it would be very helpful if you could make something like @mantine/flow as a wrapper library applying mantine's style tokens, what do you think?

If it's okay as an idea, I'd like to help develop it if you don't mind!

reactflow: https://reactflow.dev

**Comment by rtivital** [comment]
I don't use react-flow in my applications, so I'd rather not maintain it on my side. If you want to create a library and share it with the community, you can do that with the extension template – https://github.com/mantinedev/extension-template

**Comment by Kenzo-Wada** [comment]
got it!
